test/nyc-integration-snap.js

```diff
@@ -0,0 +1,187 @@
+const path = require('path')
+const os = require('os')
+
+const t = require('tap')
```

Comment:
I find this files naming a little weird; I've been tending to call my snapshot files something like "foo.snap", and I thought at first this file was your snapshot not a test file. I'd be tempted to just keep the tests in `nyc-integration.js`, but perhaps in a new `describe` block, like `describe('snapshot tests')` then once all the tests are moved over, get rid of that describe.

Reply:
@bcoe it's harder (not impossible) to use the snapshot matching with the mocha globals mode in tap, so I think this was a very reasonable approach to lower the difficulty of the move. We can definitely make more moves after this is merged

Reply:
sounds good, let's just add a TODO to this effect then.

Reply:
I've renamed the test files so that we have snapshot tests in `nyc-integration.js` and the tests which still need to be migrated are in `nyc-integration-old.js`.  This way we don't have the `snap` in the test filename.